clustering/singleton/server/src/main/java/org/wildfly/clustering/singleton/server/LegacyDistributedSingletonService.java

```diff
@@ -92,18 +89,8 @@ public T getValue() {
     }
 
     @Override
-    public CommandDispatcher<LegacySingletonContext<T>> getCommandDispatcher() {
-        return super.getCommandDispatcher();
-    }
-
-    @Override
-    public Optional<T> getLocalValue() {
-        try {
-            return this.isPrimary() ? Optional.ofNullable(this.primaryController.getValue()) : null;
-        } catch (IllegalStateException e) {
-            // This might happen if primary service has not yet started, or if node is no longer the primary node
-            return null;
-        }
+    public CommandDispatcher<GroupMember, LegacySingletonContext<T>> getCommandDispatcher() {
+        return this.get().getCommandDispatcher();
     }
 
     static class LazySupplier<T> implements Supplier<T>, Consumer<T> {
```

Comment:
With changes to the LegacyDistributedSingletonService this has become an orphan.